authLdap.php

```diff
@@ -565,9 +563,9 @@ function wp_setcookie($username, $password, $already_md5 = false, $home = '', $s
  * he isn't
  * @conf boolean authLDAP
  */
-function authLdap_show_password_fields($return, $user)
+function authLdap_show_password_fields($user)
```

Comment:
Yeah, I wondered about that.  I was getting an error where only one argument was getting passed so I made an assumption.  Probably a bad one.